### PR TITLE
PRAISE THE SUN

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Solaire
add: PRAISE THE SUN
del: PRAISE THE SUN
tweak: PRAISE THE SUN
fix: PRAISE THE SUN
wip: PRAISE THE SUN
soundadd: PRAISE THE SUN
sounddel: PRAISE THE SUN
imageadd: PRAISE THE SUN
imagedel: PRAISE THE SUN
spellcheck: PRAISE THE SUN
experiment: PRAISE THE SUN
/:cl:
praise_the_sun_by_immp-d8bfvos

### CODE REVIEW

**Comment by Armhulen** [comment]
why not

**Comment by MrPerson** [comment]
I know this is a meme, but the fact you failed trying to make an if else chain, I'm just smh tbh fam

**Comment by Iamgoofball** [comment]
this isnt actually a meme and

# HOW THE FUCK

**Comment by Bill-Luxe** [comment]
fuck you
just fuck you

**Comment by Bawhoppen** [comment]
>:+1:ing your own PR

**Comment by PKPenguin321** [comment]
in a world where reddit votes can determine if a PR stays open or not, giving yourself a :+1: is unfortunately optimal

File: code/modules/jobs/job_types/civilian_chaplain.dm [hunk 1]
```diff
@@ -65,15 +65,24 @@ Chaplain
...
+			B.throwforce = 0
+			B.SetLuminosity(9)
 	feedback_set_details("religion_name","[new_religion]")
 	ticker.Bible_name = B.name
 
 	var/new_deity = "Space Jesus"
 	if(H.client && H.client.prefs.custom_names["deity"])
 		new_deity = H.client.prefs.custom_names["deity"]
-	B.deity_name = new_deity
+	if(lowertext(new_religion) == "sun")
```
Comment (RemieRichards) [inline code/modules/jobs/job_types/civilian_chaplain.dm]: I support the feature, but not this snowflake.
add a local/proc var to the top of the proc (or somewhere before the if) called like, ```override_diety_name```, then where you do ```B.diety_name = "The Sun```  in the ```if("sun")``` section simply do ```override_diety_name = "The Sun"``` instead

**Comment by GunHog** [comment]
*Sigh*...this is what you are doing instead of working on Guncrafting...Shake my head, To be honest, Family.

The feature itself makes sense, the Chaplain already has plenty of swords and his Crusader armor is basically begging for this.

**Comment by optimumtact** [comment]
Needs to compile.

**Comment by KorPhaeron** [comment]
FINISH GUNCRAFTING

**Pull request closed without merging**